### DISCUSSION TITLE
Nur Messen-Verbraucher im Sunny-Portal über den SAE erstellen - geht das ?

### DISCUSSION THREAD

**Discussion by raldie** [camueller/SmartApplianceEnabler#259] [Installation und Betrieb]
Hallo liebe Leute,
ich benötige einmal eure Hilfe. Bei dem Ausbau der möglichst optimalen Nutzung des PV-Überschusses bin ich auf folgendes Problem gestoßen.
Bei 2 Waschmaschinen, 2 Geschirrspülern, 2 Trocknern, 2 Kühlschränken und Gefriertruhen in unserem Familienhaushalt kommt man sehr schnell an die Grenze der 12 steuerbaren Geräte im Sunny Portal.
Mit jedem Gerät (z.B. Gosund SP1) wir nun die Anzahl heruntergezählt und nun nähere ich mich folglich der restlichen 12 nicht steuerbaren Sunny-Portalgeräte.
Im Sunny Portal kann man z.B. bei der Anlage eines Verbrauchers angeben, ob nur gemessen werden soll.
Diese Verbraucher würden dann nicht in die 12 schaltbaren Verbraucher eingehen.
Das wären bei mir zur Zeit z.B. 5 Geräte für die Kühl/Gefrierschränke und Wärempumpe.

Meine Frage jetzt, kann man über den SAE einen Verbraucher (Kühlschrank, Gefrierschrank, Wärmepumpe), also alle Geräte die den Schalter 'Immer eingeschaltet' besitzen, als 'nur Messen-Verbraucher' im Sunny Portal deklarieren, um die 12 steuerbaren Geräte im Portal zu erhalten?
Würden die 'nur messen' Verbraucher in die Überschuss-Kalkulation vom Home-Manager mit einbezogen werden, oder mache ich da einen Gedankenfehler?

Danke für eure Hilfe.
LG
Ralf

**Comment by camueller** [comment]
Grundsätzlich unterstützt der Sunny Home Manager nur 12 Geräte. Die Tatsache, dass man mit der in der Dokumentation beschriebenen Vorgehensweise 22 Geräte anlegt kann, ist mehr oder weniger ein Fehler im Sunny Portal/Sunny Home Manager. Dementsprechend gibt es auch keine Möglichkeit, diese Geräte im SEMP-Protokoll unterschidlich zu markieren.

Leg die Geräte einfach an und mach Dir nicht zu viele Gedanken über die Unterscheidung zwischen steuerbar und nicht steuerbar ...
